src/main/resources/jenkins/plugins/openstack/compute/OpenStack.groovy

```diff
@@ -0,0 +1,89 @@
+package jenkins.plugins.openstack.compute
+
+import jenkins.plugins.openstack.compute.internal.DestroyMachine
+import jenkins.plugins.openstack.compute.internal.Openstack
+import org.openstack4j.model.compute.Server
+
+class OpenStack implements Serializable {
```

Comment:
I presume this is not an API for other plugins to use. Annotate with `@Restricted(NoExternalUse.class)` or `/*package*/`.